[ANY.RUN pack update] Create new integrations and playbooks

pro100broo opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
No issues

## Description
* Refactored and migrated Sandbox and Threat Intelligence Lookup functions to the SDK.
* Added new Threat Intelligence Feeds integration.
* Refactored and updated playbooks. Added new playbooks.

## Must have
- [ ] Tests
- [ ] Documentation

ilappe commented:
Thanks @pro100broo for your contribution.
Apologies for the delay
we will review it ASAP
in the while, please go over the [errors](https://github.com/demisto/content/actions/runs/15706312515/job/44252851811?pr=40283) and try to fix them
thanks in advance

pro100broo commented:
Hey, @ilappe 
Thanks for the answer.

I have fixed all critical errors according to preliminary in the last commit:
https://github.com/demisto/content/pull/40283/commits/810152fd05d1a779d82fdcaa12192863d3c1fdde

So, the rest of the errors are just linter issues. 
For example, I received the following error:
```
/home/runner/work/content/content/Packs/ANYRUN/Integrations/AnyRunLookup/AnyRunLookup.py:1:0: E9011 : test-module command is not implemented in the python file, it is essential for every integration. Please add it to your code. For more information see: https://xsoar.pan.dev/docs/integrations/code-conventions#test-module
```

But I [implemented](https://github.com/pro100broo/Cortex-XSOAR-integration/blob/810152fd05d1a779d82fdcaa12192863d3c1fdde/Packs/ANYRUN/Integrations/AnyRunLookup/AnyRunLookup.py#L80) the test-module command in this module. Most of the errors refer to similar cases.

Please, let me know if I should fix any such errors.

ilappe commented on Packs/ANYRUN/Integrations/AnyRunSandbox/AnyRunSandbox.yml:
```diff
@@ -0,0 +1,1476 @@
+      name: anyrun-detonate-file-windows
+      description: 'Perform File analysis using Windows VM.'
+      outputs:
+        - contextPath: ANYRUN.TaskID
```
Using a generic and identical key in the context of several different commands may make it difficult for a user to distinguish between the results of different runs of different commands.
it should be some unique context path for each command

ilappe commented on Packs/ANYRUN/Integrations/AnyRunSandbox/AnyRunSandbox.yml:
are you sure this command have a context output
from the code it's like there is no context output under this context path

ilappe reviewed (changes requested):
Great work!
please see my initial review comments
thanks in advance!

pro100broo commented on Packs/ANYRUN/Integrations/AnyRunSandbox/AnyRunSandbox.py:
```diff
@@ -0,0 +1,367 @@
+        indicator_type = get_indicator_type(indicator)
+        indicator_score = convert_indicator_score(reputation)
+
+        payload = {
+            "indicator": {
+                "indicator_type": indicator_type,
+                "value": indicator.get('ioc'),
+                "score": score,
+                "vendor": "ANY.RUN",
+                "CustomFields": {
+                    "vendor": "ANY.RUN",
+                    "service": "ANY.RUN Cloud Sandbox",
+                    "description": f"https://app.any.run/tasks/{task_uuid}"
+                },
+                "investigationIDs": [incident_id]
+            }
+        }
+
+        make_api_call(params, 'POST', 'indicator/create', payload)
+        indicators.append({'type': indicator_type, 'value': indicator.get('ioc'), 'verdict': indicator_score})
+
```
We need to link each indicator to a specific incident as it is part of the investigation. So,  I can't do it using CommandResults

DeanArbel commented on Packs/ANYRUN/pack_metadata.json:
```diff
@@ -1,27 +1,26 @@
-        "X3",
-        "X5",
-        "ENT_PLUS"
+        "marketplacev2"
     ]
```
@pro100broo I suggest returning the platform + supportedModules, as it will make it available for the corresponding marketplaces in the new PANW platform.
This is a part of an ongoing effort to migrate existing content packs to the new PANW Platform with supported modules. Unfortunately, we do not yet have public documents about this. however, these modules will cover the corresponding marketplaces above.

pro100broo replied:
@DeanArbel Thank you for the answer. So, I'll just return the old version of this fields

merit-maita commented:
Hello @pro100broo 
i will continue with you with this contribution, please handle the merge conflicts, if the build passes i will approve it.

merit-maita commented:
> @merit-maita I fixed all critical issues, according the pre-commit summary. Can you please check pre-commit logs?
> 
> I found one error but I don't understand how to fix it. Can you please give me some advice?
> 
> ```shell
> markdownlint-cli2........................................................Failed
> - hook id: markdownlint-cli2
> - duration: 0.85s
> - files were modified by this hook
> 
> markdownlint-cli2 v0.17.2 (markdownlint v0.37.4)
> Finding: Packs/ANYRUN/Integrations/AnyRunLookup/README.md Packs/ANYRUN/Integrations/AnyRunFeeds/README.md Packs/ANYRUN/Integrations/AnyRunSandbox/README.md Packs/ANYRUN/README.md
> ```

it's fine, i'll handle it in the internal pr

merit-maita commented:
@MLainer1  can you go over this pr as it's TIM related?
@Benimanela do you approve it?

pro100broo commented:
Hello, @MosheEichler 
Let me know when you finish the review. I will correct all the errors.

pro100broo commented:
> > Hello, @MosheEichler Let me know when you finish the review. I will correct all the errors.
> 
> From my side it's OK You need approve from @merit-maita and @Benimanela

OK, thanks

pro100broo commented:
Hey @merit-maita @Benimanela 
Can we continue the review process?

qfortress-dev reviewed (changes requested):
Hi @pro100broo, Thank you for the work on this contribution. The implementation looks good overall.
I have only one comment from a security perspective at this stage:

### General
- You set the `fromversion` to `8.0.0`. Is there a specific reason for this?  
  - If not, please change it to `6.0.0` to ensure users with on-prem version 6 instances can install the integration.
- Please make sure to use `demisto-sdk format` to standardize the files.

### Playbook
- Please update the XSOAR generic playbook references (`Detonate File - Generic`, `Detonate URL - Generic v1.5`) to use the **new playbooks** instead of the deprecated playbooks.

Let me know once you've addressed the feedback so I can review again!

pro100broo commented:
Hey, @merit-maita 
I resolved all critical issues, according the pre-commit summary. Can you please check?

merit-maita commented:
@pro100broo the validation step is still failing, you can see all errors in read under
"validate.................................................................Failed" section
for example, the unit tests file is missing for the feed, some of the feed's params are not defined as expected.

pro100broo commented on Packs/ANYRUN/ReleaseNotes/2_0_0.md:
```diff
@@ -0,0 +1,63 @@
+- New: Added a new integration- ANY.RUN TI Lookup. TI Lookup is a searchable database of IOCs, IOAs, IOBs, and events for threat hunting and a service for browsing malicious files by their content.
+
+
+- Added the following commands:
```
This is simply the output of the release notes generation script. I can delete this section.

pro100broo commented:
> > Hi, @Benimanela Can you please review the latest changes?
> 
> Thank you for addressing the previous feedback and for updating the generic playbooks.
> 
> * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
>   
>   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
>   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> 
> Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!

Hi, @Benimanela
1. Yes, we can improve this by adding a step to select the operating system.
2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.

merit-maita commented:
> RM116

you're right, please keep the name of the image as it is (with _ instead of a point)
and add this line the pack_ignore file of the common playbooks pack:
[file:playbook-Detonate_URL_-_Generic_v1.5.yml]
ignore=RM116

merit-maita commented on Packs/ANYRUN/ReleaseNotes/2_0_0.md:
```diff
@@ -0,0 +1,59 @@
+
+- Deprecated. Use ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux, ANYRUN Detonate Url Android instead.
+
+##### Deprecated: Detonate File From URL - ANYRUN
```
```suggestion
##### Detonate File From URL - ANYRUN (Deprecated)
```

merit-maita commented on Packs/ANYRUN/ReleaseNotes/2_0_0.md:
```diff
@@ -0,0 +1,59 @@
+
+- Deprecated. Use ANYRUN Detonate File Windows, ANYRUN Detonate File Linux, ANYRUN Detonate File Android instead.
+
+##### Deprecated: Detonate URL - ANYRUN
```
```suggestion
##### Detonate URL - ANYRUN (Deprecated)
```

merit-maita commented on Packs/ANYRUN/ReleaseNotes/2_0_0.md:
```diff
@@ -0,0 +1,59 @@
+
+#### Integrations
+
+##### Deprecated: ANY.RUN
```
```suggestion
##### ANY.RUN (Deprecated)
```

pro100broo commented:
Hey, @merit-maita 
It looks like your [latest requested changes](https://github.com/demisto/content/pull/40283#discussion_r2213074451) in the release notes are not being accepted.

Let me know if I did something wrong

```
Closing graph.
Packs/ANYRUN/ReleaseNotes/2_0_0.md: [RN114] - The following release note headers are invalid:
Content items: Playbooks: Detonate File - ANYRUN (Deprecated), Detonate File From URL - ANYRUN (Deprecated), Detonate URL - ANYRUN (Deprecated), Integrations: ANY.RUN (Deprecated)

Validate summary
The following errors were thrown as a part of this pr: RN114.
The following errors can be ignored: RN114.
If the AG100 validation in the pre-commit GitHub Action fails, the pull request cannot be force-merged.
The following errors don't run as part of the nightly flow and therefore can be force merged: RN114.
```

merit-maita commented:
> Hey, @merit-maita It looks like your [latest requested changes](https://github.com/demisto/content/pull/40283#discussion_r2213074451) for the release notes are not being accepted.
> 
> Please, let me know if I did something wrong.
> 
> ```
> Closing graph.
> Packs/ANYRUN/ReleaseNotes/2_0_0.md: [RN114] - The following release note headers are invalid:
> Content items: Playbooks: Detonate File - ANYRUN (Deprecated), Detonate File From URL - ANYRUN (Deprecated), Detonate URL - ANYRUN (Deprecated), Integrations: ANY.RUN (Deprecated)
> 
> Validate summary
> The following errors were thrown as a part of this pr: RN114.
> The following errors can be ignored: RN114.
> If the AG100 validation in the pre-commit GitHub Action fails, the pull request cannot be force-merged.
> The following errors don't run as part of the nightly flow and therefore can be force merged: RN114.
> ```

you can ignore it, i'll fix it in the internal pr.

merit-maita commented:
@Benimanela do you approve it?

Benimanela commented:
> > > > > Hi, @Benimanela Can you please review the latest changes?
> > > > 
> > > > 
> > > > Thank you for addressing the previous feedback and for updating the generic playbooks.
> > > > 
> > > > * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
> > > >   
> > > >   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
> > > >   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> > > > 
> > > > Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!
> > > 
> > > 
> > > Hi, @Benimanela
> > > 
> > > 1. Yes, we can improve this by adding a step to select the operating system.
> > > 2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.
> > > 
> > > Please tell me if there is anything I need to fix or improve in the current PR
> > 
> > 
> > HI @pro100broo, Did you do these changes?
> 
> Hi, @Benimanela
> 
> Can you please answer my [last question](https://github.com/demisto/content/pull/40283#issuecomment-3077938342)?
> 
> Regarding the points I mentioned: Yes, I can add a new step to select the operating system.
> 
> The following playbooks use the `File.EntryID` field to retrieve the file to analyse. ANYRUN Detonate File Android, ANYRUN Detonate File Windows, ANYRUN Detonate File Linux
> 
> The following playbooks use the `URL.Data` field to retrieve the url to analyse. ANYRUN Detonate Url Android, ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux
> 
> Is the logic correct or do I need to fix something?

Thanks for the details! I just want to clarify:

- Can the three detonate playbooks for URL or file each handle **all** file types, or is each one designed for specific file types/operating systems?
- If each playbook is meant for a particular file type or OS, we should add a step before the detonation that checks the input and routes it to the correct playbook.
- If the playbooks are interchangeable and can all process any file/URL, then the current setup is fine.

Let me know which is the case so we can make sure the playbook logic is correct!

pro100broo commented:
> > > > > > Hi, @Benimanela Can you please review the latest changes?
> > > > > 
> > > > > 
> > > > > Thank you for addressing the previous feedback and for updating the generic playbooks.
> > > > > 
> > > > > * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
> > > > >   
> > > > >   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
> > > > >   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> > > > > 
> > > > > Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!
> > > > 
> > > > 
> > > > Hi, @Benimanela
> > > > 
> > > > 1. Yes, we can improve this by adding a step to select the operating system.
> > > > 2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.
> > > > 
> > > > Please tell me if there is anything I need to fix or improve in the current PR
> > > 
> > > 
> > > HI @pro100broo, Did you do these changes?
> > 
> > 
> > Hi, @Benimanela
> > Can you please answer my [last question](https://github.com/demisto/content/pull/40283#issuecomment-3077938342)?
> > Regarding the points I mentioned: Yes, I can add a new step to select the operating system.
> > The following playbooks use the `File.EntryID` field to retrieve the file to analyse. ANYRUN Detonate File Android, ANYRUN Detonate File Windows, ANYRUN Detonate File Linux
> > The following playbooks use the `URL.Data` field to retrieve the url to analyse. ANYRUN Detonate Url Android, ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux
> > Is the logic correct or do I need to fix something?
> 
> Thanks for the details! I just want to clarify:
> 
> * Can the three detonate playbooks for URL or file each handle **all** file types, or is each one designed for specific file types/operating systems?
> * If each playbook is meant for a particular file type or OS, we should add a step before the detonation that checks the input and routes it to the correct playbook.
> * If the playbooks are interchangeable and can all process any file/URL, then the current setup is fine.
> 
> Let me know which is the case so we can make sure the playbook logic is correct!

Thank you for the detailed explanation. So I changed the playbooks, can you please check?

Here are my suggestions for changes:
### Add new inputs group for the ANY.RUN playbooks
<img width="485" height="823" alt="image" src="https://github.com/user-attachments/assets/def97203-1086-4763-934f-ded9a7242483" />

### Add two new steps before executing playbooks
<img width="780" height="373" alt="image" src="https://github.com/user-attachments/assets/d5f3920e-35b7-4079-a56e-5169907dd8ff" />

These steps are similar for file and url playbooks

pro100broo commented:
> > > > > > > > Hi, @Benimanela Can you please review the latest changes?
> > > > > > > 
> > > > > > > 
> > > > > > > Thank you for addressing the previous feedback and for updating the generic playbooks.
> > > > > > > 
> > > > > > > * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
> > > > > > >   
> > > > > > >   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
> > > > > > >   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> > > > > > > 
> > > > > > > Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!
> > > > > > 
> > > > > > 
> > > > > > Hi, @Benimanela
> > > > > > 
> > > > > > 1. Yes, we can improve this by adding a step to select the operating system.
> > > > > > 2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.
> > > > > > 
> > > > > > Please tell me if there is anything I need to fix or improve in the current PR
> > > > > 
> > > > > 
> > > > > HI @pro100broo, Did you do these changes?
> > > > 
> > > > 
> > > > Hi, @Benimanela
> > > > Can you please answer my [last question](https://github.com/demisto/content/pull/40283#issuecomment-3077938342)?
> > > > Regarding the points I mentioned: Yes, I can add a new step to select the operating system.
> > > > The following playbooks use the `File.EntryID` field to retrieve the file to analyse. ANYRUN Detonate File Android, ANYRUN Detonate File Windows, ANYRUN Detonate File Linux
> > > > The following playbooks use the `URL.Data` field to retrieve the url to analyse. ANYRUN Detonate Url Android, ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux
> > > > Is the logic correct or do I need to fix something?
> > > 
> > > 
> > > Thanks for the details! I just want to clarify:
> > > 
> > > * Can the three detonate playbooks for URL or file each handle **all** file types, or is each one designed for specific file types/operating systems?
> > > * If each playbook is meant for a particular file type or OS, we should add a step before the detonation that checks the input and routes it to the correct playbook.
> > > * If the playbooks are interchangeable and can all process any file/URL, then the current setup is fine.
> > > 
> > > Let me know which is the case so we can make sure the playbook logic is correct!
> > 
> > 
> > Thank you for the detailed explanation. So I changed the playbooks, can you please check?
> > Here are my suggestions for changes:
> > ### Add new inputs group for the ANY.RUN playbooks
> > <img alt="image" width="490" height="827" src="https://private-user-images.githubusercontent.com/85073445/468411951-b56e4309-9f0c-40d7-9540-d61afc0b1711.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE5NTEtYjU2ZTQzMDktOWYwYy00MGQ3LTk1NDAtZDYxYWZjMGIxNzExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyOWU1YjQwMWRhODE4NDYzOWI0NzI2ODMyYTM5Y2QzM2IyNzE4ODk5ZDE2MDQxMmQyYTk5MGNkOTY2YmJiOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tQO4sHiqUjQMaWZEhqvjRYvlP9ywxBme2l2_S_erU9s">
> > ### Add two new steps before executing playbooks
> > <img alt="image" width="780" height="373" src="https://private-user-images.githubusercontent.com/85073445/468411497-d5f3920e-35b7-4079-a56e-5169907dd8ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE0OTctZDVmMzkyMGUtMzViNy00MDc5LWE1NmUtNTE2OTkwN2RkOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4Y2NjNmU3NmJhNzk1NjIzOWM5NDcwOGRhNDBkNzVjYjIxOTk3NTkxZDE3OWM3OWYyOGNiY2UzNjc0OTFhOTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7U86XxdbMRjEQ8u506_e1qLjyqHnmdFGaBu3iZGilPA">
> > These steps are similar for file and url playbooks
> 
> Hi,
> 
> Great, this looks good the flow is almost perfect now. Just a couple of small changes:
> 
> * In the playbook **Detonate URL - Generic v1.5**, please add the missing Android playbook, and remove the link from the above steps to the ANY.RUN playbooks.
>   ![Detonate URL - Generic v1.5](https://private-user-images.githubusercontent.com/112556158/468556259-deaee214-818d-4f1f-a6f5-a3f379721771.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU2MjU5LWRlYWVlMjE0LTgxOGQtNGYxZi1hNmY1LWEzZjM3OTcyMTc3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MWEzNjdkODdkMDI1MjFjNTVkNGE4NTQ2ZTlmMjUxZjk4YTFlODc5MmVhOWFjMDM3YTYwOWU4YzAxMWEwNTZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZRbRsI-d-urqI2qKt8t8FUHgZYATHr7fX5EfDDcooT8)
> * Also, please fix the same in **Detonate File - Generic**:
>   ![Issue Screenshot](https://private-user-images.githubusercontent.com/112556158/468559101-fe4fbdfc-17ca-43e6-b887-8a325f619772.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU5MTAxLWZlNGZiZGZjLTE3Y2EtNDNlNi1iODg3LThhMzI1ZjYxOTc3Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGIxNjU5MWYzNWY0MzcyY2M5YzUxODE5N2U5MTM3NTY5MmEzMDQxNmQwMjBlNzIzZDQ0NjU2MGQyMGMwYWJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.74pUGHWgHwjf4fsBEEvuusKArLTi9LPcD2TRMHzdKgM)
> 
> Let me know once these changes are made so I can review again!

Hi, @Benimanela 
Im so sorry.  It looks like this is a playbook export error on my side. I fixed it

Benimanela commented:
> > > > > > > > > Hi, @Benimanela Can you please review the latest changes?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Thank you for addressing the previous feedback and for updating the generic playbooks.
> > > > > > > > 
> > > > > > > > * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
> > > > > > > >   
> > > > > > > >   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
> > > > > > > >   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> > > > > > > > 
> > > > > > > > Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!
> > > > > > > 
> > > > > > > 
> > > > > > > Hi, @Benimanela
> > > > > > > 
> > > > > > > 1. Yes, we can improve this by adding a step to select the operating system.
> > > > > > > 2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.
> > > > > > > 
> > > > > > > Please tell me if there is anything I need to fix or improve in the current PR
> > > > > > 
> > > > > > 
> > > > > > HI @pro100broo, Did you do these changes?
> > > > > 
> > > > > 
> > > > > Hi, @Benimanela
> > > > > Can you please answer my [last question](https://github.com/demisto/content/pull/40283#issuecomment-3077938342)?
> > > > > Regarding the points I mentioned: Yes, I can add a new step to select the operating system.
> > > > > The following playbooks use the `File.EntryID` field to retrieve the file to analyse. ANYRUN Detonate File Android, ANYRUN Detonate File Windows, ANYRUN Detonate File Linux
> > > > > The following playbooks use the `URL.Data` field to retrieve the url to analyse. ANYRUN Detonate Url Android, ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux
> > > > > Is the logic correct or do I need to fix something?
> > > > 
> > > > 
> > > > Thanks for the details! I just want to clarify:
> > > > 
> > > > * Can the three detonate playbooks for URL or file each handle **all** file types, or is each one designed for specific file types/operating systems?
> > > > * If each playbook is meant for a particular file type or OS, we should add a step before the detonation that checks the input and routes it to the correct playbook.
> > > > * If the playbooks are interchangeable and can all process any file/URL, then the current setup is fine.
> > > > 
> > > > Let me know which is the case so we can make sure the playbook logic is correct!
> > > 
> > > 
> > > Thank you for the detailed explanation. So I changed the playbooks, can you please check?
> > > Here are my suggestions for changes:
> > > ### Add new inputs group for the ANY.RUN playbooks
> > > <img alt="image" width="490" height="827" src="https://private-user-images.githubusercontent.com/85073445/468411951-b56e4309-9f0c-40d7-9540-d61afc0b1711.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE5NTEtYjU2ZTQzMDktOWYwYy00MGQ3LTk1NDAtZDYxYWZjMGIxNzExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyOWU1YjQwMWRhODE4NDYzOWI0NzI2ODMyYTM5Y2QzM2IyNzE4ODk5ZDE2MDQxMmQyYTk5MGNkOTY2YmJiOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tQO4sHiqUjQMaWZEhqvjRYvlP9ywxBme2l2_S_erU9s">
> > > ### Add two new steps before executing playbooks
> > > <img alt="image" width="780" height="373" src="https://private-user-images.githubusercontent.com/85073445/468411497-d5f3920e-35b7-4079-a56e-5169907dd8ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE0OTctZDVmMzkyMGUtMzViNy00MDc5LWE1NmUtNTE2OTkwN2RkOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4Y2NjNmU3NmJhNzk1NjIzOWM5NDcwOGRhNDBkNzVjYjIxOTk3NTkxZDE3OWM3OWYyOGNiY2UzNjc0OTFhOTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7U86XxdbMRjEQ8u506_e1qLjyqHnmdFGaBu3iZGilPA">
> > > These steps are similar for file and url playbooks
> > 
> > 
> > Hi,
> > Great, this looks good the flow is almost perfect now. Just a couple of small changes:
> > 
> > * In the playbook **Detonate URL - Generic v1.5**, please add the missing Android playbook, and remove the link from the above steps to the ANY.RUN playbooks.
> >   ![Detonate URL - Generic v1.5](https://private-user-images.githubusercontent.com/112556158/468556259-deaee214-818d-4f1f-a6f5-a3f379721771.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU2MjU5LWRlYWVlMjE0LTgxOGQtNGYxZi1hNmY1LWEzZjM3OTcyMTc3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MWEzNjdkODdkMDI1MjFjNTVkNGE4NTQ2ZTlmMjUxZjk4YTFlODc5MmVhOWFjMDM3YTYwOWU4YzAxMWEwNTZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZRbRsI-d-urqI2qKt8t8FUHgZYATHr7fX5EfDDcooT8)
> > * Also, please fix the same in **Detonate File - Generic**:
> >   ![Issue Screenshot](https://private-user-images.githubusercontent.com/112556158/468559101-fe4fbdfc-17ca-43e6-b887-8a325f619772.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU5MTAxLWZlNGZiZGZjLTE3Y2EtNDNlNi1iODg3LThhMzI1ZjYxOTc3Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGIxNjU5MWYzNWY0MzcyY2M5YzUxODE5N2U5MTM3NTY5MmEzMDQxNmQwMjBlNzIzZDQ0NjU2MGQyMGMwYWJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.74pUGHWgHwjf4fsBEEvuusKArLTi9LPcD2TRMHzdKgM)
> > 
> > Let me know once these changes are made so I can review again!
> 
> Hi, @Benimanela Im so sorry. It looks like this is a playbook export error on my side. I fixed it

Looks good now!

- Please update the condition names to "Windows", "Linux", and "Android" to match the corresponding target playbook.
  ![Condition Names](https://github.com/user-attachments/assets/56076687-59c0-41b4-981d-3c9017a994e0)

- Also, please align the playbook steps using the "Auto Align" button for better readability.
  ![Auto Align](https://github.com/user-attachments/assets/46b6232e-a5cb-4e53-9922-2bcf74744c06)

pro100broo commented:
> > > > > > > > > > Hi, @Benimanela Can you please review the latest changes?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Thank you for addressing the previous feedback and for updating the generic playbooks.
> > > > > > > > > 
> > > > > > > > > * I noticed that you updated the playbooks to use your three specific detonation playbooks (Android, Linux, Windows).
> > > > > > > > >   
> > > > > > > > >   * Can we further improve this by adding a step that dynamically chooses the correct playbook based on the file type or operating system, instead of having all three as options?
> > > > > > > > >   * What logic are you using to determine which detonate playbook (Android, Linux, Windows) should be run for a given input? Is there an indicator (such as file extension, user input, or a field in the incident) that can help automate this decision within the playbook?
> > > > > > > > > 
> > > > > > > > > Let me know your thoughts or if you can enhance the playbook flow to make this selection more dynamic!
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Hi, @Benimanela
> > > > > > > > 
> > > > > > > > 1. Yes, we can improve this by adding a step to select the operating system.
> > > > > > > > 2. I designed new playbooks to run inside the incident context. This way, the user can select necessary playbook using the incident settings. If there is a need to run the playbook in a different scenario, the user will need to customize the input fields.
> > > > > > > > 
> > > > > > > > Please tell me if there is anything I need to fix or improve in the current PR
> > > > > > > 
> > > > > > > 
> > > > > > > HI @pro100broo, Did you do these changes?
> > > > > > 
> > > > > > 
> > > > > > Hi, @Benimanela
> > > > > > Can you please answer my [last question](https://github.com/demisto/content/pull/40283#issuecomment-3077938342)?
> > > > > > Regarding the points I mentioned: Yes, I can add a new step to select the operating system.
> > > > > > The following playbooks use the `File.EntryID` field to retrieve the file to analyse. ANYRUN Detonate File Android, ANYRUN Detonate File Windows, ANYRUN Detonate File Linux
> > > > > > The following playbooks use the `URL.Data` field to retrieve the url to analyse. ANYRUN Detonate Url Android, ANYRUN Detonate Url Windows, ANYRUN Detonate Url Linux
> > > > > > Is the logic correct or do I need to fix something?
> > > > > 
> > > > > 
> > > > > Thanks for the details! I just want to clarify:
> > > > > 
> > > > > * Can the three detonate playbooks for URL or file each handle **all** file types, or is each one designed for specific file types/operating systems?
> > > > > * If each playbook is meant for a particular file type or OS, we should add a step before the detonation that checks the input and routes it to the correct playbook.
> > > > > * If the playbooks are interchangeable and can all process any file/URL, then the current setup is fine.
> > > > > 
> > > > > Let me know which is the case so we can make sure the playbook logic is correct!
> > > > 
> > > > 
> > > > Thank you for the detailed explanation. So I changed the playbooks, can you please check?
> > > > Here are my suggestions for changes:
> > > > ### Add new inputs group for the ANY.RUN playbooks
> > > > <img alt="image" width="490" height="827" src="https://private-user-images.githubusercontent.com/85073445/468411951-b56e4309-9f0c-40d7-9540-d61afc0b1711.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE5NTEtYjU2ZTQzMDktOWYwYy00MGQ3LTk1NDAtZDYxYWZjMGIxNzExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyOWU1YjQwMWRhODE4NDYzOWI0NzI2ODMyYTM5Y2QzM2IyNzE4ODk5ZDE2MDQxMmQyYTk5MGNkOTY2YmJiOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tQO4sHiqUjQMaWZEhqvjRYvlP9ywxBme2l2_S_erU9s">
> > > > ### Add two new steps before executing playbooks
> > > > <img alt="image" width="780" height="373" src="https://private-user-images.githubusercontent.com/85073445/468411497-d5f3920e-35b7-4079-a56e-5169907dd8ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMwOTE1NDcsIm5iZiI6MTc1MzA5MTI0NywicGF0aCI6Ii84NTA3MzQ0NS80Njg0MTE0OTctZDVmMzkyMGUtMzViNy00MDc5LWE1NmUtNTE2OTkwN2RkOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIxVDA5NDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4Y2NjNmU3NmJhNzk1NjIzOWM5NDcwOGRhNDBkNzVjYjIxOTk3NTkxZDE3OWM3OWYyOGNiY2UzNjc0OTFhOTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7U86XxdbMRjEQ8u506_e1qLjyqHnmdFGaBu3iZGilPA">
> > > > These steps are similar for file and url playbooks
> > > 
> > > 
> > > Hi,
> > > Great, this looks good the flow is almost perfect now. Just a couple of small changes:
> > > 
> > > * In the playbook **Detonate URL - Generic v1.5**, please add the missing Android playbook, and remove the link from the above steps to the ANY.RUN playbooks.
> > >   ![Detonate URL - Generic v1.5](https://private-user-images.githubusercontent.com/112556158/468556259-deaee214-818d-4f1f-a6f5-a3f379721771.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU2MjU5LWRlYWVlMjE0LTgxOGQtNGYxZi1hNmY1LWEzZjM3OTcyMTc3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MWEzNjdkODdkMDI1MjFjNTVkNGE4NTQ2ZTlmMjUxZjk4YTFlODc5MmVhOWFjMDM3YTYwOWU4YzAxMWEwNTZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZRbRsI-d-urqI2qKt8t8FUHgZYATHr7fX5EfDDcooT8)
> > > * Also, please fix the same in **Detonate File - Generic**:
> > >   ![Issue Screenshot](https://private-user-images.githubusercontent.com/112556158/468559101-fe4fbdfc-17ca-43e6-b887-8a325f619772.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDE3NjAsIm5iZiI6MTc1MzEwMTQ2MCwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NTU5MTAxLWZlNGZiZGZjLTE3Y2EtNDNlNi1iODg3LThhMzI1ZjYxOTc3Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxMjM3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGIxNjU5MWYzNWY0MzcyY2M5YzUxODE5N2U5MTM3NTY5MmEzMDQxNmQwMjBlNzIzZDQ0NjU2MGQyMGMwYWJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.74pUGHWgHwjf4fsBEEvuusKArLTi9LPcD2TRMHzdKgM)
> > > 
> > > Let me know once these changes are made so I can review again!
> > 
> > 
> > Hi, @Benimanela Im so sorry. It looks like this is a playbook export error on my side. I fixed it
> 
> Looks good now!
> 
> * Please update the condition names to "Windows", "Linux", and "Android" to match the corresponding target playbook.
>   ![Condition Names](https://private-user-images.githubusercontent.com/112556158/468647354-56076687-59c0-41b4-981d-3c9017a994e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDcwMzEsIm5iZiI6MTc1MzEwNjczMSwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NjQ3MzU0LTU2MDc2Njg3LTU5YzAtNDFiNC05ODFkLTNjOTAxN2E5OTRlMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxNDA1MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOTdkYzc0MzkyNzFkNTNjNmM2ZDg2OTIwZmY4OWQzMDhlZGFkNWY0MzQzZGY0MWVmODBiMjc4MWI1ZjhhOTZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jC_cM24lFUkeYIkRVdtuwxYCLcwFLw38jV3jrgCyjno)
> * Also, please align the playbook steps using the "Auto Align" button for better readability.
>   ![Auto Align](https://private-user-images.githubusercontent.com/112556158/468647197-46b6232e-a5cb-4e53-9922-2bcf74744c06.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxMDcwMzEsIm5iZiI6MTc1MzEwNjczMSwicGF0aCI6Ii8xMTI1NTYxNTgvNDY4NjQ3MTk3LTQ2YjYyMzJlLWE1Y2ItNGU1My05OTIyLTJiY2Y3NDc0NGMwNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyMVQxNDA1MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTk4YTg2ODY2OTFhMDFjYmQwMzUzNWVjMmVmMmExZWRhMzIyOTYzYWRhMzU4MDJmZDU0OThjOTA5YjI1OTg2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.E7jLk1A2VvB4R1NJBaOJ2H7iQfxjb1We5iG4JsPDQxg)

Its done!

Benimanela reviewed (approved):
Approved from my end

pro100broo commented:
Hi, @merit-maita 
Is there anything else required from my side to continue the process?
